src/options.cpp

```diff
@@ -152,16 +162,27 @@ options parse_options(int argc, char** argv) {
   options o;
 
   std::optional<int> next_vim_file_bufnr;
+  bool has_stdin = false;
 
   arg_parser parser(argc, argv);
   while (!parser.done()) {
     if (const char* argument = parser.match_argument()) {
-      if (std::strcmp(argument, "-") == 0) {
-        o.stdinput = true;
-        continue;
+      if (argument == "-"sv) {
+        if (has_stdin) {
+          continue;
```

Comment:
Hmm, perhaps we should issue a warning if `-` or `--stdin` is given multiple times.

Reply:
i thought it would be unnecessary noisy, done.